Makefile

```diff
@@ -584,10 +615,17 @@ docopen: $(apidocs_html)
 docclean:
 	$(RM) -r out/doc
 
+# Prepare the build for running the tests
+# Note that this is not run by the CI jobs on ARM or Linux with FIPS
 build-ci:
 	$(PYTHON) ./configure $(CONFIG_FLAGS)
 	$(MAKE)
 
+# Run by most testing CIs, exceptions are:
+#  - ARM (Raspberry Pis): the binaries are cross-compiled elsewhere
+#    before being used to run tests. See test-ci-native.
+#  - Linux with FIPS: the build is done differently, the tests are still
```

Comment:
fips isn't an exception here, _although_ it does run `test-ci` directly a second time with FIPS enabled

coverage is another exception that doesn't run this